le_utils/resources/presetlookup.json

```diff
@@ -254,7 +254,7 @@
 	"h5p_thumbnail": {
 		"readable_name": "H5P Thumbnail",
 		"multi_language": false,
-		"supplementary": true,
+		"supplementary": false,
```

Comment:
Don't need to update this value - just the one change you have made above is sufficient!

Reply:
Okay cool I will update this